benchmark_runner.py

```diff
@@ -96,9 +111,19 @@
             influence_factory = RPSLJEFactory(ihvp_mode=args.ihvp_mode,
                                               start_layer=args.start_layer,
                                               dataset_hessian_size=args.dataset_hessian_size,
-                                              n_cgd_iters=args.n_cgd_iters,
+                                              n_opt_iters=args.n_opt_iters,
                                               feature_extractor=args.feature_extractor
                                               )
+        elif method_name == 'boundary_weights':
```

Comment:
It is a lot of conditional statement, maybe we should consider refactor it at some point..